ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/**
+ `CommandBar` is a horizontal scrollable list of icon buttons divided by groups.
+ Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItem` and `trailingItem` add fixed buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
+ Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
+ */
+@objc(MSFCommandBar)
+open class CommandBar: UIView {
+    // Hierarchy:
+    //
+    // leadingButton
+    // containerView
+    // |--layer.mask -> containerMaskLayer (fill containerView)
+    // |--subviews
+    // |  |--scrollView (fill containerView)
+    // |  |  |--subviews
+    // |  |  |  |--stackView
+    // |  |  |  |  |--buttons (fill scrollView content)
+    // trailingButton
+
+    // MARK: - Public methods
+
+    public init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
```

Comment:
I don't see this public initializer with ObjC usage, looks like something is missing to make it available? (I cloned and built this PR branch locally to test out the UI)

Reply:
Ah, so this is missing @objc, seems some other places are also. I will comment where I find these missing, thanks!